packages/tools/viewer-alpha/public/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Viewer Local Development</title>
+
+        <style>
+            html,
+            body {
+                width: 100%;
+                height: 100%;
+                padding: 0;
+                margin: 0;
+                overflow: hidden;
+            }
+        </style>
+    </head>
+
+    <body>
+        <babylon-viewer src="https://playground.babylonjs.com/scenes/BoomBox.glb" env="https://assets.babylonjs.com/environments/ulmerMuenster.env"> </babylon-viewer>
+        <script src="webtest.js"></script>
+        <script>
+            (async () => {
+                // "BabylonViewer" is defined as the "library" in webpack.config.js
+                BabylonViewer.registerCustomElements();

```

Comment:
nit - if you want to use modules instead, you can `import { BabylonViewer } from ...`, instead of populating the global namespace

Reply:
Talked offline, we will revisit using the [HtmlWebpackPlugin](https://webpack.js.org/plugins/html-webpack-plugin/) later to see if we can solve this problem and also improve the debugging experience for any inline JS in the html file. For now it's a small amount of code so we will leave it as is.